packages/graphql-server/package.json

```diff
@@ -23,36 +23,33 @@
   },
   "dependencies": {
     "@babel/runtime-corejs3": "7.20.6",
-    "@envelop/depth-limit": "1.8.0",
-    "@envelop/disable-introspection": "3.6.0",
-    "@envelop/filter-operation-type": "3.6.0",
-    "@envelop/parser-cache": "4.7.0",
-    "@envelop/validation-cache": "4.7.0",
+    "@envelop/core": "3.0.4",
+    "@envelop/depth-limit": "2.0.4",
+    "@envelop/disable-introspection": "4.0.4",
+    "@envelop/filter-operation-type": "4.0.4",
```

Comment:
we've had several outstanding renovate PRs for upgrading these to their latest majors. since graphql-yoga uses envelop/core v3, it's time we upgrade them, so I've integrated them here:
- https://github.com/redwoodjs/redwood/pull/6679
- https://github.com/redwoodjs/redwood/pull/6680
- https://github.com/redwoodjs/redwood/pull/6681